.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -133,6 +133,7 @@ jobs:
 
           python3.11 -m pip install instructlab-training[cuda]
```

Comment:
(No action required) I think installation of `instructlab` package with `cuda` deps is now done as `pip install .[cuda]` since https://github.com/instructlab/instructlab/pull/1753 was merged.